### PR TITLE
Add cascade lake partition for csd3

### LINKED ISSUE #186: csd3 skylake retired
TODO: Remove skylake partition and update to cascade lake

https://docs.hpc.cam.ac.uk/hpc/user-guide/cclake.html

### PR DESCRIPTION
**Pull request by tkoskela**
This PR copies the skylake partition into a cascade lake partition. Only update is to the number of cores per node. This works in my initial tests but probably needs further checking.

Closes #186

### CODE REVIEW

File: benchmarks/reframe_config.py [hunk 1]
```diff
@@ -113,6 +113,30 @@ def spack_root_to_path():
                 },
             ]
         },  # end CSD3 Icelake
+        {
+            # https://www.hpc.cam.ac.uk/systems/peta-4
```
Comment (giordano) [inline benchmarks/reframe_config.py:117]: I get a 404 error page

Comment (ma595) [reply benchmarks/reframe_config.py]: https://www.hpc.cam.ac.uk/high-performance-computing

**Comment by ma595** [comment]
Is anything holding up this merge? I am happy to test it if necessary.

**Comment by tkoskela** [comment]
Should be good to go. Would be great if you could test! Unless @giordano has more comments?

**Comment by giordano** [comment]
No, it's good for me if tests confirm it's alright.

**Comment by tkoskela** [comment]
We have automation that deletes merged branches? cool!

**Comment by giordano** [comment]
It's a [setting in GitHub](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches)

**Pull request merged**